app/models/event.rb

```diff
@@ -181,6 +181,11 @@ def blame
     "Something wrong with this listing? Contact #{name} <#{email}> with reference {url}"
   end
 
+  def og_title
+    str = "#{summary}, #{date}, #{time}"
```

Comment:
What happens if an event doesn't have a time? Or do they always?

Reply:
They always do, we do have an open ticket for proper all day event support tho somewhere.